loops/julia/code.jl

```diff
@@ -0,0 +1,12 @@
+let
+    a = zeros(Int,10^4)
+    u = parse(Int,ARGS[1])
+    r = only(rand(1:10^4,1))
+    @inbounds for i ∈ 1:10^4
+        @inbounds for j ∈ 1:10^5
+            a[i] = a[i] + u
```

Comment:
You are not doing the module operation here, see the other languages.

Reply:
You are right, I missed that!